### PR TITLE
workflows/tests: untap mongodb on macOS.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This will enable `brew audit` to pass.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/flow.rb [hunk 1]
```diff
@@ -20,10 +20,6 @@ class Flow < Formula
   uses_from_macos "rsync" => :build
   uses_from_macos "unzip" => :build
 
-  on_linux do
-    depends_on "elfutils"
```
Comment (MikeMcQuaid) [inline Formula/flow.rb:24]: @iMichka Note that this will need removed because it doesn't exist in homebrew-core.

File: Formula/freerdp.rb [hunk 2]
```diff
@@ -25,7 +25,6 @@ class Freerdp < Formula
   on_linux do
     depends_on "ffmpeg"
     depends_on "glib"
-    depends_on "systemd"
```
Comment (MikeMcQuaid) [inline Formula/freerdp.rb:28]: @iMichka Note that this will need removed because it doesn't exist in homebrew-core.

File: Formula/flow.rb (hunk 1 above)
Comment (iMichka) [reply Formula/flow.rb:24]: No issue, it was added by mistake.

I need to either port these formulae to mac, or start adding them as Linux-only: but in this case I will also add minimal Linux-Ci to make sure it is tested. I’ll start on working on this in September

Comment (MikeMcQuaid) [reply Formula/flow.rb:24]: Cool, thanks. Don't worry about CI, it'll be caught in future by the new CI as-of this PR.

**Pull request merged**